src/payment/store.rs

```diff
@@ -302,6 +302,8 @@ pub enum PaymentKind {
 		txid: Txid,
 		/// The confirmation status of this payment.
 		status: ConfirmationStatus,
+		/// The on-chain fees that were paid for this payment.
+		fee_msat: u64,
```

Comment:
Any reason we track fees for on-chain payments but not for lightning payments? I suppose we could for outbound lightning payments, but IIUC we can't know for inbound payments.

Also, should we use sats here instead of msats? Or is this to align with msats used in `PaymentDetails`?

Reply:
LGTM. Seems it would be useful to include.